client/manager_test_helpers.go

```diff
@@ -90,7 +90,7 @@ func TestHelperCreateGetUpdateDeleteClient(k string, m Storage) func(t *testing.
 			Contacts:                          []string{"aeneas1", "aeneas2"},
 			SecretExpiresAt:                   0,
 			SectorIdentifierURI:               "https://sector",
-			JSONWebKeys:                       &x.JoseJSONWebKeySet{JSONWebKeySet: &jose.JSONWebKeySet{Keys: []jose.JSONWebKey{{KeyID: "foo", Key: []byte("asdf"), Certificates: []*x509.Certificate{}}}}},
+			JSONWebKeys:                       &x.JoseJSONWebKeySet{JSONWebKeySet: &jose.JSONWebKeySet{Keys: []jose.JSONWebKey{{KeyID: "foo", Key: []byte("asdf"), Certificates: []*x509.Certificate{}, CertificateThumbprintSHA1: []uint8{}, CertificateThumbprintSHA256: []uint8{}}}}},
```

Comment:
for failed tests

```
time="2020-05-27T16:53:11+09:00" level=info msg="No tracer configured - skipping tracing setup"
--- FAIL: TestManagers (6.25s)
    driver.go:152: beginning to wait
    test_helper.go:191: Skipping Docker setup because environment variable TEST_DATABASE_MYSQL is set to: mysql://root:secret@(127.0.0.1:3444)/mysql?parseTime=true&multiStatements=true
    test_helper.go:143: Skipping Docker setup because environment variable TEST_DATABASE_POSTGRESQL is set to: postgres://postgres:secret@127.0.0.1:3445/postgres?sslmode=disable
    test_helper.go:244: Skipping Docker setup because environment variable TEST_DATABASE_COCKROACHDB is set to: cockroach://root@127.0.0.1:3446/defaultdb?sslmode=disable
    driver.go:149: cddb done
    driver.go:139: Pg done
    driver.go:144: myssql done
    driver.go:154: done waiting
    clean_sql.go:12: Cleaning up database: postgres
    clean_sql.go:12: Cleaning up database: postgres
    clean_sql.go:12: Cleaning up database: mysql
    clean_sql.go:40: Successfully cleaned up database: mysql
    clean_sql.go:40: Successfully cleaned up database: postgres
    clean_sql.go:40: Successfully cleaned up database: postgres
    driver.go:102: clean and migrate done
    driver.go:102: clean and migrate done
    driver.go:102: clean and migrate done
    manager_test.go:44: connected
    --- FAIL: TestManagers/case=create-get-update-delete#01 (0.03s)
        --- FAIL: TestManagers/case=create-get-update-delete#01/db=postgres (0.03s)
            manager_test_helpers.go:213:
                	Error Trace:	manager_test_helpers.go:213
                	            				manager_test_helpers.go:127
                	Error:      	Not equal:
                	            	expected: &jose.JSONWebKeySet{Keys:[]jose.JSONWebKey{jose.JSONWebKey{Key:[]uint8{0x61, 0x73, 0x64, 0x66}, KeyID:"foo", Algorithm:"", Use:"", Certificates:[]*x509.Certificate{}, CertificatesURL:(*url.URL)(nil), CertificateThumbprintSHA1:[]uint8(nil), CertificateThumbprintSHA256:[]uint8(nil)}}}
                	            	actual  : &jose.JSONWebKeySet{Keys:[]jose.JSONWebKey{jose.JSONWebKey{Key:[]uint8{0x61, 0x73, 0x64, 0x66}, KeyID:"foo", Algorithm:"", Use:"", Certificates:[]*x509.Certificate{}, CertificatesURL:(*url.URL)(nil), CertificateThumbprintSHA1:[]uint8{}, CertificateThumbprintSHA256:[]uint8{}}}}

                	            	Diff:
                	            	--- Expected
                	            	+++ Actual
                	            	@@ -12,4 +12,6 @@
                	            	    CertificatesURL: (*url.URL)(<nil>),
                	            	-   CertificateThumbprintSHA1: ([]uint8) <nil>,
                	            	-   CertificateThumbprintSHA256: ([]uint8) <nil>
                	            	+   CertificateThumbprintSHA1: ([]uint8) {
                	            	+   },
                	            	+   CertificateThumbprintSHA256: ([]uint8) {
                	            	+   }
                	            	   }
                	Test:       	TestManagers/case=create-get-update-delete#01/db=postgres
FAIL
coverage: 5.2
FAIL	github.com/ory/hydra/client	7.061s
FAIL
exit status 1
make: *** [test] Error 1
```